packages/turf-standard-deviational-ellipse/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Feature, Polygon, Properties, Point } from '@turf/helpers'
+ 
+/**
+ * http://turfjs.org/docs/#standarddeviational-ellipse
+ */
+
+export default function (
+    points: Feature<Point>,
```

Comment:
Missing `FeatureCollection<Point>` to types

Reply:
Thanks. I'm still getting the hang of the typing.

Reply:
Don't sweat it 👍 It's new to a lot of folks, even I'm still trying to figure out the best way to write these definitions.